google_fonts/.travis.yml

```diff
@@ -0,0 +1,34 @@
+os:
```

Comment:
So there is already a travis.yml file in the root of this repo. I think you could probably just updated the travis script to include the google fonts project:

https://github.com/material-components/material-components-flutter-experimental/blob/master/travis_script.sh

Reply:
That worked, thanks!